frontend-v2/src/features/data/LoadData.tsx

```diff
@@ -190,24 +190,43 @@ const LoadData: FC<ILoadDataProps> = ({ state, firstTime }) => {
       >
         {showData && <SetUnits state={state} firstTime={firstTime} />}
       </Box>
-      <Box style={style.dropAreaContainer}>
-        <Box {...getRootProps({ style: style.dropArea })}>
-          <input {...getInputProps()} />
-          <Typography>
-            Drag &amp; drop some files here, or click to select files
-          </Typography>
+      {!showData && (
```

Comment:
If I upload the wrong file by mistake, then `showData` will be true so I won't be able to correct my mistake and upload the correct file. Is that intentional?

Reply:
Yes. According to the mockups, there will be an additional button added next to 'Previous/Next' called 'Cancel' - which will allow users to cancel working on current file and upload a new one. On user cancelation showData prop will be set back to false for the dropdown to appear.

Mockups - https://www.figma.com/design/bxDmCsAM7vIFpe6mBBJ9p2/Pathfinder?node-id=457-82&node-type=canvas&t=lGqrIgh9gTjLUPPY-0 - Desktop - S1 - option1 and the one below showcasing the tooltip

Reply:
Cool! In that case this looks good. My only comment is that it's forbidden to embed one clickable element inside another clickable element in HTML, so the button inside the clickable div might be invalid.

Reply:
That's a valid point, will update it on Monday before merging.